knowledge-base/datagrid-selection-column.md

```diff
@@ -24,11 +24,11 @@ To implement such a column and to avoid the usage of the templates, use the [Cel
 
 Here is the implementation:
 
-**1.** Draw a `CheckBox` using the `CellRenderer` approach. 
+1. Draw a `CheckBox` using the `CellRenderer` approach. 
 
-    **1.1** Create a custom `CheckboxColumnRenderer` class that inherits from `DataGridCellRenderer`. 
+    1.1 Create a custom `CheckboxColumnRenderer` class that inherits from `DataGridCellRenderer`. 
```

Comment:
Double-check that this works, because I don't think the 1.1 is recognizable markdown and this will be treated as a list in the output.